ods_ci/tests/Tests/1300__model_registry/1302_model_registry_smoke.robot

```diff
@@ -68,7 +78,6 @@ Remove Model Registry Non UI
 
 Run Curl Command And Verify Response
     [Documentation]    Runs a curl command to verify response from server
-    [Arguments]    ${URL}
```

Comment:
I'm not sure why this change is part of this PR meaning that we'll prefer the global variable instead of passing it here. I'm not gonna block as I can see that in this file there is quite chaotic approach - some of the keywords use argument for this, some use global value... this isn't a good approach from my point of view.